debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlDefaultValueConverter.java

```diff
@@ -62,7 +62,7 @@ public class MySqlDefaultValueConverter implements DefaultValueConverter {
     @Immutable
     private static final Set<Integer> TRIM_DATA_TYPES = Collect.unmodifiableSet(Types.TINYINT, Types.INTEGER,
             Types.DATE, Types.TIMESTAMP, Types.TIMESTAMP_WITH_TIMEZONE, Types.TIME, Types.BOOLEAN, Types.BIT,
-            Types.NUMERIC, Types.DECIMAL, Types.FLOAT, Types.DOUBLE, Types.REAL);
+            Types.NUMERIC, Types.DECIMAL, Types.FLOAT, Types.DOUBLE, Types.REAL, Types.BIGINT, Types.SMALLINT);
```

Comment:
Is now `TRIM_DATA_TYPES` a supreset of complete `NUMBER_DATA_TYPES`? If it is it probably makes sense to not repeat all the values but instead either make `TRIM_DATA_TYPES` of `NUMBER_DATA_TYPES` and the additional values or remove numeric types from this var and modify `convert` method to use both attributes.

Reply:
Thanks for the suggestions. I have updated the PR and enriched the test.